docs/modules/hive/pages/usage.adoc

```diff
@@ -173,17 +179,19 @@ metastore:
       config:
         resources:
           requests:
-            cpu: 200m
-            memory: 2Gi
+            cpu: "250m"
+            memory: "512Mi"
           limits:
-            cpu: "4"
-            memory: 2Gi
+            cpu: "1000m"
+            memory: "512Mi"
           storage:
             data:
-              capacity: 2Gi
+              capacity: 512Mi
 ----
 
-WARNING: The default values are _most likely_ not sufficient to run a proper cluster in production. Please adapt according to your requirements.
+The operator may configure an additional container for log aggregation. This is done when log aggregation is configured as described in xref:concepts:logging.adoc[]. The resources for this container cannot be configured using the mechanism described above. Use pod overrides for this purpose.
```

Comment:
`podOverrides`.
Please link to the concepts page, section podOverrides, I think you need to use the nightly version